eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -136,20 +145,15 @@ case class Data(nodes: Map[PublicKey, NodeAnnouncement],
                 // for which we have not yet received an 'end' message
                )
 
+// @formatter:off
 sealed trait State
-
 case object NORMAL extends State
 
 case object TickBroadcast
-
 case object TickPruneStaleChannels
-
+case object TickNetworkStats
```

Comment:
This should be a verb:

```suggestion
case object TickComputeNetworkStats
```